src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: Roboto, Arial, sans-serif;` is good, but you could add more alternatives like `font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;`.